packages/xo-web/src/common/xo/index.js

```diff
@@ -615,6 +615,38 @@ export const addHostsToPool = pool =>
     })
   })
 
+export const enableAdvancedLiveTelemetry = async host => {
+  const isConfiguredToReceiveStreaming = await _call(
+    'netdata.isConfiguredToReceiveStreaming',
+    { host: host.id }
+  )
+  if (!isConfiguredToReceiveStreaming) {
+    await _call('netdata.configureXoaToReceiveData', {})
+  }
+  await _call('netdata.configureHostToStreamHere', {
+    host: host.id,
+  })
+  success(_('advancedLiveTelemetry'), _('enableAdvancedLiveTelemetrySuccess'))
+}
+
+export const isNetDataInstalledOnHost = async host => {
+  const isNetDataInstalledOnHost = await _call(
+    'netdata.isNetDataInstalledOnHost',
+    { host: host.id }
+  )
+  const hostApiKey = await _call('netdata.getHostApiKey', {
+    host: host.id,
+  })
+  const localApiKey = await _call('netdata.getLocalApiKey', {})
+  if (
+    Boolean(isNetDataInstalledOnHost) &&
+    hostApiKey.trim() === localApiKey.trim()
```

Comment:
Should be done in the `netdata` plugin, not here.

Reply:
I'll open a merge request for this in `xoa project`

Reply:
https://gitlab.com/vates/xoa/merge_requests/37